features/0429-prepare-req-rich-pres/README.md

```diff
@@ -0,0 +1,63 @@
+# 0429: Prerequisites to Request Rich Presentation
+- Authors: [Brent Zundel](<brent.zundel@evernym.com>), [Ken Ebert](<ken@sovrin.org>)
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2020-02-21
+- Status Note: Part of proposed Rich Schema capabilities for credentials 
+- Supersedes: 
+- Start Date: 2020-02-19 
+- Tags: [feature](/tags.md#feature), [rich-schemas](/tags.md#rich-schemas)
+
+## Summary
+
+Describes the prerequisites an issuer must ensure are in place before requesting
+a rich presentation.
+
+## Motivation
+
+To inform verifiers of the steps they should take in order to make sure they
+have the necessary rich schema objects in place before they use them to request
+proofs.
+
+## Tutorial
+
+###Rich Schema Presentation Definition Workflow 
+1. The verifier checks his wallet or the ledger to see if the presentation
+definition already exists. (The verifier determines which attribute or
+predicates he needs a holder to present to satisfy the verifier's business
+rules. Presentation definitions specify desired attributes and predicates).
+   1. If not, the verifier creates a new presentation definition and stores the
+   presentation definition in his wallet locally and, optionally, anchors it to
+   the verifiable data registry. (Anchoring the presentation definition to the
```

Comment:
Should we clarify what we mean by anchoring here? Is it writing the full presentation definition's content to the ledger, or just an anchor/digital_fingerprint/hash of the full content?

Reply:
I added a comment that explains anchoring as being possible with either strategy.